src/functionalTest/java/uk/gov/hmcts/reform/em/hrs/DownloadHearingRecordingScenarios.java

```diff
@@ -18,6 +19,8 @@
 
     @Autowired
     private TestUtil testUtil;
+    private CcdDataStoreApiClient ccdClient;
```

Comment:
I don't think this is needed, isn't CcdDataStoreApiClient already autowired in the baseTest?